plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueExpressionUtil.java

```diff
@@ -147,6 +147,12 @@ static Optional<String> buildGlueExpressionForSingleDomain(String columnName, Do
             return Optional.empty();
         }
 
+        // Glue throws an exception (e.g. input string: "__HIVE_D" is not an integer)
```

Comment:
can we add corresponding tests for the quoted types null filter effectiveness?

Reply:
Added some types. Skipped `varbinary` type because it's unsupported as a partition.